backend/repositories/match-repository.js

```diff
@@ -22,8 +22,32 @@ const storeMatch = async (match) => {
     try {
         result = await storageContext.insertMatch(match)
 
-        await updateRatingForTeam(winningTeam, match.ratingChange, storageContext)
-        await updateRatingForTeam(losingTeam, -match.ratingChange, storageContext)
+        // FIXME: foosbot's report of rating change is incorrect if the number of players on each side is not the same
+        // one player beat two players
+        if (winningTeam.length < losingTeam.length) {
```

Comment:
This function is called storeMatch and its only responsibility is to store the match, not to do some computing logic. i'd edit edit the logic of whatever generates the match to do this.

maybe augment the match with two properties:
{team1RatingChange:..., team2RatingChange:...}